lib/view/disclaimer_view.dart

```diff
@@ -1,206 +1,202 @@
-import 'dart:io';
-
+import 'package:intl/intl.dart';
 import 'package:flutter/material.dart';
 import 'package:shared_preferences/shared_preferences.dart';
-import '../hard_data.dart';
+import '../models/disclaimer_model.dart';
 import '../routes.dart';
-import '../style.dart';
+import '../strings.dart';
+import '../styles.dart';
 
+/// Disclaimer screen presented on app startup until the user agrees to the disclaimer
 class DisclaimerView extends StatefulWidget {
   @override
   _DisclaimerViewState createState() => _DisclaimerViewState();
 }
 
-class _DisclaimerViewState extends State<DisclaimerView>
-    with SingleTickerProviderStateMixin {
-  final String _title = 'Disclaimer and conditions of use';
+class _DisclaimerViewState extends State<DisclaimerView> {
+  @override
+  void initState() {
+    super.initState();
+  }
```

Comment:
If you are not doing anything in the initState(), I would remove it.

Reply:
Good point ... I was playing with something in there. I usually keep initState and dispose in there regardless .... but thats just me. I'll remove 👍